### PR TITLE
Refactor FXIOS-5460 [v111] Update theming in ContextualHintViewController

### PR DESCRIPTION
**Pull request by thatswinnie**
[FXIOS-5460](https://mozilla-hub.atlassian.net/browse/FXIOS-5460)
#12730

Simulator Screen Shot - iPhone 14 Pro - 2023-01-27 at 10 36 42
Simulator Screen Shot - iPhone 14 Pro - 2023-01-27 at 10 38 49
Simulator Screen Shot - iPhone 14 Pro - 2023-01-27 at 10 40 27

### CODE REVIEW

**Comment by thatswinnie** [comment]
Build passed
Screenshot 2023-01-27 at 13 20 28

File: Client/Frontend/Contextual Hint/ContextualHintViewController.swift [hunk 1]
```diff
@@ -125,6 +127,7 @@ class ContextualHintViewController: UIViewController, OnViewDismissable {
     override func viewDidLoad() {
         super.viewDidLoad()
         commonInit()
+        applyTheme()
```
Comment (lmarceau) [inline Client/Frontend/Contextual Hint/ContextualHintViewController.swift:130]: There should be a `listenForThemeChange` as well

**Comment by thatswinnie** [comment]
Build passed: 
Screenshot 2023-01-30 at 11 03 24

**Comment by cwzilla** [comment]
@thatswinnie Looking good! I may need to modify the gradient background colour in the future (as currently in the lightest parts, the text is not accessible). I'll look into this and create a new ticket later on if we decide to change the colours. Thanks :)

**Pull request merged**